src/ws/login.js

```diff
@@ -653,13 +653,12 @@ if (window.NodeList && !NodeList.prototype.forEach)
                                                             key_type ? " -t " + key_type : "");
 
         id("hostkey-fingerprint").textContent = data.default;
-        var key = data["host-key"];
```

Comment:
This was duplicated above. ESLint complained, so I removed it here. https://github.com/cockpit-project/cockpit/blob/e4898065560fba768585f8637eb3b30f2d2c34c8/src/ws/login.js#L632

Reply:
Oh, nice!